src/components/Books/Book.js

```diff
@@ -1,30 +1,45 @@
 import React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- I would advise you to solve the Linter manually. Do not disable linters because it contributes to clean code, and avoid bugs in your code.

Reply:
Thanks for the insight fixed.